src/schema/v2/__tests__/fair.test.js

```diff
@@ -301,22 +275,42 @@ describe("Fair", () => {
               hasNextPage
               endCursor
             }
+            pageCursors {
+              around {
+                page
+              }
+              first {
+                page
+              }
+              previous {
+                page
+              }
+            }
           }
         }
       }
     `
 
     const data = await runQuery(query, context)
-    expect(data).toEqual({
+
+    const {
       fair: {
-        shows: {
-          pageInfo: {
-            endCursor: "1234567890",
-            hasNextPage: true,
-          },
-        },
+        shows: { pageCursors, pageInfo },
       },
+    } = data
+
+    const { around, first, previous } = pageCursors
+
+    expect(around.length).toBe(4)
+    expect(first).toBe(null)
+    expect(previous).toBe(null)
+    expect(pageInfo).toEqual({
```

Comment:
**minor, non-blocking:** @anastasiapyzhik do you think we should add tests where we expect multiple pages? for example, we could try `first:1`, or perhaps`first:1`, get a result back, pull out a cursor, and then do `first: 1, after: cursor` to see more of these cursor fields being populated. Let me know what you think!

Reply:
@mdole updated tests so you can see it now 🙂